app/controllers/api/v0/admins_controller.rb

```diff
@@ -0,0 +1,32 @@
+module Api
+  module V0
+    class AdminsController < BaseController
+      def add_portfolio
+        portfolio = Portfolio.create!(portfolio_params)
+        render json: portfolio
+      end
+
+      def add_portfolio_item_to_portfolio
+        portfolio = Portfolio.find_by(id: params[:portfolio_id])
```

Comment:
This should not be `find_by`, it should raise an exception if not found and render a 404.

Reply:
See what I did [here](https://github.com/ManageIQ/topological_inventory-api/blob/master/app/controllers/api/mixins/update_mixin.rb#L4)

Reply:
Also, use Strong Params